src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -5128,7 +5144,10 @@ def create_functionapp(cmd, resource_group_name, name, storage_account, plan=Non
     if (plan_info is not None and is_plan_elastic_premium(cmd, plan_info)) or consumption_plan_location is not None:
         site_config.app_settings.append(NameValuePair(name='WEBSITE_CONTENTAZUREFILECONNECTIONSTRING',
                                                       value=con_string))
-        site_config.app_settings.append(NameValuePair(name='WEBSITE_CONTENTSHARE', value=_get_content_share_name(name)))
+        content_share_name = _get_content_share_name(name)
```

Comment:
I think this function may cause the dismatch between resource name and the recording file.
https://github.com/Azure/azure-cli/blob/c84cd74c330f16f85e9bc1826da3e7efd7f781d1/src/azure-cli/azure/cli/command_modules/appservice/custom.py#L5425-L5430

Reply:
Hi @yanzhudd We actually do intend to create a new content share resource with a new name on behalf of the customer.  Is there any way to work around this? Or should we just make the tests run live?

Reply:
yes, you can mark the tests as live only, and please remove the related recording files as we..

Reply:
Thank you @yanzhudd Those tests are already marked as live_only and there are no recording files for them.